### PR TITLE
Completed typescript-data-model exercise - Shea Pone

### PR DESCRIPTION
**Pull request by SheaPone**
typescript-data-model

### CODE REVIEW

**Review by hesam — CHANGES_REQUESTED** [review]
Great so far. 

- Can you work it out so each order uses an array to represent all items purchased in it? That would be superior and flexible to using an optional 'secondPartofOrder'  property?

**Comment by SheaPone** [comment]
Screenshot 2024-07-12 at 7 40 52 PM

File: typescript-data-model/order-history.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+interface amazonItem {
```
Comment (hesam) [inline typescript-data-model/order-history.ts:1]: By convention user defined types/interfaces should be capitalized (as opposed to primitive types). So I'd go with: AmazonItem

**Review by hesam — APPROVED** [review]
Great all around!

**Pull request merged**